cmd/bedMaxWig/bedMaxWig.go

```diff
@@ -12,14 +12,27 @@ import (
 	"log"
 )
 
-func bedMaxWig(infile string, database string, chromsizeFile string, outfile string) {
+func bedMaxWig(infile string, database string, chromsizeFile string, outfile string, norm bool) {
 	var records []bed.Bed = bed.Read(infile)
 	var wigData []wig.Wig = wig.Read(database)
 	var sizes []chromInfo.ChromInfo = chromInfo.ReadToSlice(chromsizeFile)
 	var outlist []bed.Bed
 	var currentBed bed.Bed = records[0]
 	var chromSlice []float64
 	var i int
+	var wigTotal float64 = 0
+
+	if norm == true { //TODO: This assumes fixedStep. Either should check for fixedStep or be able to handle both cases (fixedStep and variableStep)
+		for i := 0; i < len(wigData); i++ { //Goal here is to cycle through all the "chromosomes" of the wig: wig[0], wig[1], etc.
+			var wigCounterByChrom float64 = 0
+			for k := 0; k < len(wigData[i].Values); k++ { //Cycle through each value in the float64[]
+				var chromValueMultiplyByStep float64 = 0
+				chromValueMultiplyByStep = (float64(wigData[i].Step) * wigData[i].Values[k]) // multiply each value by the step for that chrom
+				wigCounterByChrom = wigCounterByChrom + chromValueMultiplyByStep
```

Comment:
Not a needed change, but just FYI you can also do this as 
`wigCounterByChrom += chromValueMultiplyByStep`

Reply:
Ahhh, totally forgot about this move! Will remember for next time!